### PR TITLE
EMBR-4336 updated docs for RN 5.0 release

### PR DESCRIPTION
**Pull request by jpmunz**
Re-opening this update for the React Native 5.0 release against the new public embrace-docs repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This website is built using [Docusaurus](https://docusaurus.io/), a modern static documentation generator written using React Native and is published to [https://embrace.io/docs/](https://embrace.io/docs/)
 
+Contributions are welcomed, no matter how small!
```
Comment (jpmunz) [inline README.md:5]: @vitaliyf I'm gonna revert this commit on this branch as it conflicts with main now, looks like @davidlawrencer also updated the README so wasn't sure which should be taken

Comment (vitaliyf) [reply README.md]: Yea use his - I made this one to test deploys from your branch.

Comment (davidlawrencer) [reply README.md]: Screenshot 2024-09-18 at 1 29 41 PM
@jpmunz can you clean up my typo at the end here 🙏

Comment (jpmunz) [reply README.md]: @davidlawrencer I think best if you just open a separate PR for that, even if it's a small change helps keep things a bit more organized

**Review by facostaembrace — APPROVED** [review]

**Pull request merged**